test/e2e/privileged_test.go

```diff
@@ -73,4 +74,17 @@ var _ = Describe("Podman privileged container tests", func() {
 		Expect(capAmp[1]).To(Equal(capEff[1]))
 	})
 
+	It("podman non-privileged should have very few devices", func() {
+		session := podmanTest.Podman([]string{"run", "busybox", "ls", "-l", "/dev"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(len(session.OutputToStringArray())).To(Equal(18))
```

Comment:
Can we safely guarantee this is 18, or should we say under 20?

Reply:
I want to see if 18 is firm.  Lets leave it there for now and see what happens.